helm/reana/README.md

```diff
@@ -61,6 +61,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `ingress.enabled`                                        | Create an ingress resource to access the REANA instance from outside the cluster     | true                                            |
 | `kubernetes_jobs_memory_limit`                           | Maximum default memory limit for user job containers. Exceeding this limit will terminate the container. Please see the following URL for possible values https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | 4Gi |
 | `kubernetes_jobs_max_user_memory_limit`                  | Maximum custom memory limit that users can assign to their job containers via `kubernetes_memory_limit` hint in `reana.yaml`. Exceeding this limit will terminate the container. Please see the following URL for possible values https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | None |
+| `kubernetes_jobs_timeout_limit`                          | Default timeout for user's job in *seconds*. Exceeding this time will terminate the job. Please see the following URL for more details https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup. | 604800 |
```

Comment:
```suggestion
| `kubernetes_jobs_timeout_limit`                          | Default timeout for user's jobs in *seconds*. Exceeding this time will terminate the job. Please see the following URL for more details https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup. | 604800 |
```

User's job**s** might be better? Otherwise, it seems that a user can only have a single job, but it might be just my impression, what do you think?

Reply:
Would it make sense to add "(7 days)" next to the default value? Having just a big number is not very meaningful at first sight. As an admin, I'd appreciate knowing how much this is quick.

Reply:
> Would it make sense to add "(7 days)" next to the default value?

Yes, it would. My only small worry is that it will be clear for admins that the number to enter is "604800" and not "7 days". Cause, as for now, they are not interchangeable. Entering "7 days" will not work. Although, in the description, I mentioned "*seconds*" so should be good.

 I will change it cause, I agree, a big number out of nowhere is not user-friendly.

Reply:
*question:* would it make sense to point to our documentation page instead of Kubernetes docs?

Reply:
> > Would it make sense to add "(7 days)" next to the default value?
> 
> Yes, it would. My only small worry is that it will be clear for admins that the number to enter is "604800" and not "7 days". Cause, as for now, they are not interchangeable. Entering "7 days" will not work. Although, in the description, I mentioned "_seconds_" so should be good.
> 
> I will change it cause, I agree, a big number out of nowhere is not user-friendly.

We can add it as part of the "Description" instead of as part of the "Default" column, right after the k8s URL. What do you think?